coffee/chosen.jquery.coffee

```diff
@@ -44,7 +44,7 @@ class Chosen extends AbstractChosen
     if @is_multiple
       @container.html '<ul class="chosen-choices"><li class="search-field"><input type="text" value="' + @default_text + '" class="default" autocomplete="off" style="width:25px;" /></li></ul><div class="chosen-drop"><ul class="chosen-results"></ul></div>'
     else
-      @container.html '<a class="chosen-single chosen-default"><span>' + @default_text + '</span><div><b></b></div></a><div class="chosen-drop"><div class="chosen-search"><input type="text" autocomplete="off" /></div><ul class="chosen-results"></ul></div>'
+      @container.html '<a class="chosen-single chosen-default"><span>' + @default_text + '</span><div></div></a><div class="chosen-drop"><div class="chosen-search"><input type="text" autocomplete="off" placeholder="Filter" /></div><ul class="chosen-results"></ul></div>'
```

Comment:
1. The `<b> </b>` was unnecessary, as I've moved the sprite image right onto that parent `<div> </div>`.
2. I've added a placeholder of "Filter" to this input. Watching people use Chosen in Harvest we've seen them skip right over the input. We should probably make this customizable, but maybe in another PR.